### PR TITLE
TC_01.01.006 | Sign in to Jerkins > Sign in with invalid password

### PR DESCRIPTION
**Pull request by KateKazakovaJS**
https://trello.com/c/yUHWu6dv/327-tc0101006-sign-in-to-jerkins-sign-in-with-invalid-password

### CODE REVIEW

**Review by npkazakova — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1847,6 +2374,17 @@
...
+      "version": "0.1.0",
+      "resolved": "https://registry.npmjs.org/yocto-queue/-/yocto-queue-0.1.0.tgz",
+      "integrity": "sha512-rVksvsnNCdJ/ohGc6xgPwyN8eheCxsiLM8mxuE/t/mOVqJewPuO1miLpTHQiRgTKCLexL4MeAFVagts7HmNZ2Q==",
+      "engines": {
+        "node": ">=10"
+      },
+      "funding": {
+        "url": "https://github.com/sponsors/sindresorhus"
+      }
     }
```
Comment (RuslanG13) [inline package-lock.json:2388]: Changes in configuration files are not allowed
`!!Do not push changed files as:

package.json package.lock.json cypress.config.js e2e.js ci.yml cancel.yml globalHooks.js cleanData.js`

**Review by RuslanG13 — CHANGES_REQUESTED** [review]

**Review by RuslanG13 — CHANGES_REQUESTED** [review]
Changes in configuration files are not allowed
`!!Do not push changed files as:

package.json package.lock.json cypress.config.js e2e.js ci.yml cancel.yml globalHooks.js cleanData.js`

File: package-lock.json [hunk 2]
```diff
@@ -70,24 +71,24 @@
       }
     },
     "node_modules/@types/node": {
-      "version": "14.18.63",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-14.18.63.tgz",
-      "integrity": "sha512-fAtCfv4jJg+ExtXhvCkCqUKZ+4ok/JQk01qDKhL5BDDoS3AxKXhV5/MAVUZyQnSEd2GT92fkgZl0pz0Q0AzcIQ=="
+      "version": "16.18.61",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-16.18.61.tgz",
+      "integrity": "sha512-k0N7BqGhJoJzdh6MuQg1V1ragJiXTh8VUBAZTWjJ9cUq23SG0F0xavOwZbhiP4J3y20xd6jxKx+xNUhkMAi76Q=="
```
Comment (RuslanG13) [inline package-lock.json:76]: Changes in configuration files are not allowed
`!!Do not push changed files as:

package.json package.lock.json cypress.config.js e2e.js ci.yml cancel.yml globalHooks.js cleanData.js`

**Review by RuslanG13 — CHANGES_REQUESTED** [review]

**Review by Rom-T — CHANGES_REQUESTED** [review]
You cannot make changes to the project config files. Please correct

**Review by Rom-T — APPROVED** [review]

**Review by RuslanG13 — APPROVED** [review]

**Pull request merged**